custom/icds_reports/ucr/data_sources/dashboard/commcare_user_cases.json

```diff
@@ -38,6 +38,22 @@
           "property_name": "owner_id"
         }
       },
+      {
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "expression": {
+          "datatype": null,
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "property_name",
+            "property_name": "owner_id"
+          }
```

Comment:
missing comma

Reply:
Fixed it, I will install a JSON validator on my IDE